core/src/main/java/nl/nn/adapterframework/receivers/PullingListenerContainer.java

```diff
@@ -136,17 +138,18 @@ public boolean isLongLived() {
 
 		@Override
 		public void run() {
-			log.debug(receiver.getLogPrefix()+" taskExecutor ["+ToStringBuilder.reflectionToString(taskExecutor)+"]");
+			ThreadContext.push(ClassUtils.nameOf(receiver) + " ["+receiver.getName()+"]");
```

Comment:
Het lijkt er op dat je dit opneemt om van de getLogPrefix() af te komen. 
Dan moet je wel ook een %x in de log4j conversion patterns opnemen.

Reply:
Wij gebruiken %TC een eigen implementatie om de NDC stack weer te geven. Zie `IbisNdcPatternConverter`